storageClient/storageClient.go

```diff
@@ -62,9 +62,9 @@ func (c *StorageClient) ReadNote(ctx context.Context, noteID string, eacpParams
 	return result, err
 }
 
-func (c *StorageClient) Challenge(ctx context.Context, noteID string) ([]string, error) {
+func (c *StorageClient) Challenge(ctx context.Context, noteID string, body ChallengeRequestBody) ([]string, error) {
```

Comment:
nit: for consistency with other methods in this file and module this should be more like `params ChallenengeRequest`